core/tests/unittests/test_parallel_local_folding.py

```diff
@@ -47,8 +55,8 @@ def _construct_dummy_fold_strategy(num_jobs, model_base_cls=AbstractModel, time_
         oof_pred_proba=np.array([]),
         oof_pred_model_repeats=np.array([]),
         save_folds=True,
-        num_cpus=ResourceManager.get_cpu_count(),
-        num_gpus=ResourceManager.get_gpu_count(),
+        num_cpus=NUM_CPU,
```

Comment:
Why did we move away from calling the method ?

Reply:
The cpu number needs to be mocked, otherwise the results can vary depending on the Github runner machine type.